cms/static/js/views/settings/main.js

```diff
@@ -228,6 +237,18 @@ var DetailsView = ValidatingView.extend({
                 this.setField(event);
             }
             break;
+        case 'course-title':
+            this.setField(event);
+            break;
+        case 'course-subtitle':
+            this.setField(event);
+            break;
+        case 'course-duration':
+            this.setField(event);
+            break;
+        case 'course-description':
+            this.setField(event);
+            break;
```

Comment:
Switch/case statements have a "fallthrough" feature which can be used here to save lines:
`        case 'course-title':
            this.setField(event);
            break;
        case 'course-subtitle':
            this.setField(event);
            break;
        case 'course-duration':
            this.setField(event);
            break;
        case 'course-description':
            this.setField(event);
            break;`

can be easily and safely combined into:
`        case 'course-title':
        case 'course-subtitle':
        case 'course-duration':
        case 'course-description':
            this.setField(event);
            break;`

I see other fields which can be combined into a similar pattern. Will reduce a lot of lines.